ps/jenkins/percona-server-for-mysql-8.0.groovy

```diff
@@ -119,8 +119,25 @@ def installDependencies(def nodeName) {
 }
 
 def runPlaybook(def nodeName) {
+    
+    def ps80_install_pkg_minitests_playbook
 
-    def ps80_install_pkg_minitests_playbook = 'ps_80.yml'
+    if ("${PS8_RELEASE_VERSION}") { // Checks if PS8_RELEASE_VERSION is not empty
+        echo "MAJOR VERSION IS ${PS8_RELEASE_VERSION}"
+
+        if("${PS8_RELEASE_VERSION}" == "8.0"){
+            ps80_install_pkg_minitests_playbook = 'ps_80.yml'
+            PS8_RELEASE_VERSION_PRODUCT_TO_TEST = 'ps80'
+        }
+        else if("${PS8_RELEASE_VERSION}" == "8.1" ){
+            ps80_install_pkg_minitests_playbook = 'ps_81.yml'    
+            PS8_RELEASE_VERSION_PRODUCT_TO_TEST = 'ps81'
+        }
+    }
+    else{
```

Comment:
So on each new release you are going to add additional if-block?
maybe we should reuse value from PS8_RELEASE_VERSION variable?